client/web/compose/src/components/ModuleFields/Configurator/validation.vue

```diff
@@ -149,6 +221,11 @@ export default {
   data () {
     return {
       loaded: false,
+      fieldConstraintExists: false,
```

Comment:
Ok since this is all regarding constraints, then group it under the same object, for example:
fieldConstraint: { total, exists, index, ruleIndex }